### ISSUE TITLE
Comments: Showing deleted comments

### ISSUE THREAD

**Issue by MattiasJay** [Skjutsgruppen/skjutsgruppen-reactnative#503]
Just as on the homepage we need to show when a message is deleted.

**Comment by shresthaKiran** [comment]
@MattiasJay  Sorrry, but we couldn't understand this issue. Are you talking about deleting comments ?

**Comment by MattiasJay** [comment]
Yes, this needs more explanation. On the current homepage when someone is deleting a comment there's a line saying "This comment was deleted by (name)". 

"Name" in this case could be the one who had first made the comment (owner of comment) and/or owner of ride or enabler of group.

**Comment by shresthaKiran** [comment]
Currently, when a user deletes the comment, we have both deleted the feed and actual comment from the database. So, if we were to implement this, then the old deleted comments won't be displayed as above.

Should we implement this only in comments or to other feeds (like offered/asked ride in a group) too ?

@MattiasJay
